python/ray/actor.py

```diff
@@ -713,12 +713,15 @@ def _remote(self,
                     f"ray.get_actor('{name}', namespace='{namespace}')")
 
         if lifetime is None:
-            detached = False
+            detached = None
         elif lifetime == "detached":
             detached = True
+        elif lifetime == "non-detached":
```

Comment:
Can we use "default" instead of non-detached (it's weird to have a - as an enum value, and "default" is more typical in Ray APIs).

Reply:
This case looks too odd if we don't use `non-detached` flag:

```python
default_actor_lifetime="detached"
actor.lifetime="default"
```

Reply:
I think it's more common to use underscore (_), instead of dash (-). 
But I think "non_detached" is kind of harder to understand. What about something like "coexist_with_creator"?

Reply:
I slightly prefer `non_detached` because it's consistent with the before.
@kfstorm @ericl CC